assets/src/edit-story/components/inspector/designInspector.js

```diff
@@ -40,11 +52,18 @@ function DesignInspector() {
 		updateSelectedElements( { properties: actualProperties } );
 	}, [ updateSelectedElements ] );
 	return (
-
 		<Wrapper>
-			{ panels.map( ( { Panel, type } ) => (
-				<Panel key={ type } deleteSelectedElements={ deleteSelectedElements } selectedElements={ selectedElements } onSetProperties={ handleSetProperties } />
-			) ) }
+			<TopPanels>
+				{ panels.map( ( { Panel, type } ) => (
```

Comment:
Am I missing something here, why is this code repeated?

Reply:
Heh, dummy test. I need long content to test the scrollbar. Removed in recent push.